docs/content/users/quickstart.md

```diff
@@ -47,6 +47,35 @@ To get started with [Backdrop](https://backdropcms.org), clone the project repos
     ddev launch
     ```
 
+## CakePHP
+
+Use a new or existing Composer project, or clone a Git repository.
+
+The CakePHP project type can be used with any CakePHP project >= 3.x, but it has been fully tested with CakePHP 5.x. DDEV automatically creates the `.env` file with the database information, email transport configuration and a random salt. If `.env` file already exists, `.env.ddev` will be created so you can take any variable and put into your `.env` file.
+
+=== "Composer"
+
+    ```bash
+    mkdir my-cakephp-app
+    cd my-cakephp-app
+    ddev config --project-type=cakephp --docroot=webroot
+    ddev composer create --prefer-dist cakephp/app:~5.0
+    ddev cake
+    ddev launch
```

Comment:
When I ran through this locally, the installation stop until I answered a prompt:

```
$ddev composer create --prefer-dist cakephp/app:~5.0
...
Created `/var/www/html/tmp/cache/views` directory
Set Folder Permissions ? (Default to Y) [Y,n]? 
```

Is there a way we can avoid this?

Reply:
I saw the same, but this is the composer plugin requesting perms. Maybe there's an arg to `ddev composer create --prefer-dist cakephp/app:~5.0` or maybe there's a `ddev composer config` that we can do in advance?

Reply:
I tried the `-n` (--no-interaction) but it still wanted me to interact. :(
Seems like a "composer" issue and not a DDEV one so not a blocker.

Reply:
isn't that the installer.php file from cakephp that is causing the prompt?

Reply:
Digging deeper .... looks like its `App\Console\Installer.php:65` calling `static::setFolderPermissions($rootDir, $io);`

Reply:
That questions happens in postInstall so every time you execute `composer install` it asks to reset permissions. 

if you execute `composer -n install` it actually reset permissions without interaction which is expected.

However it looks like `ddev composer create` do not keep the "no interaction" when composer run-script is called (See [composer-create.go](https://github.com/ddev/ddev/blob/96fd091844b9a9cf4b09196fe0963231ed1170a4/cmd/ddev/cmd/composer-create.go#L171)). 

I was doing some tests trying to get the flag and pass it but I am probably doing something wrong because even passing `--preserve-flags`, this returns false `preserveFlags, _ := cmd.Flags().GetBool("preserve-flags")` so I was not able to deal with --no-interaction flag and pass it to `run-script`

Reply:
@rfay mentioning you here in case you want to create a ticket to investigate my findings on this.